extensions/Worker.Extensions.CosmosDB/release_notes.md

```diff
@@ -3,3 +3,4 @@
 - My change description (#PR/#issue)
 -->
 - Fixed incorrect type of CosmosDBTriggerAttribute's `StartFromTime` property.
```

Comment:
This [change was already released in 4.0.1](https://github.com/Azure/azure-functions-dotnet-worker/releases/tag/cosmosdb-extension-4.0.1). So remove it?